.gitattributes

```diff
@@ -0,0 +1,4 @@
+CHANGELOG.md
+CODE_OF_CONDUCT.md
+.github
+README.md
```

Comment:
Please expand this so that, in the case somebody installs the plugin using GH (which should not be done), they don't get the tests and all the other config files. 

Check this one for example: https://github.com/infinum/eightshift-coding-standards/blob/main/.gitattributes